spec/classes/init_spec.rb

```diff
@@ -21,11 +21,22 @@
       is_expected.to contain_package('condor').that_requires('Yumrepo[condor]')
     end
 
-    it { is_expected.to contain_file('/etc/condor/condor_config.local') }
+    it { is_expected.to contain_file('/etc/condor/condor_config.local').that_requires('Package[condor]') }
 
-    it { is_expected.to contain_file('/etc/condor/config.d/docker') }
+    it { is_expected.to contain_file('/etc/condor/config.d/docker').that_requires('Package[condor]') }
 
-    it { is_expected.to contain_file('/etc/condor/config.d/schedd').with_content(%r{^DAEMON_LIST = MASTER, SCHEDD}) }
+    it do
+      is_expected.to contain_file('/etc/condor/config.d/schedd').with(
```

Comment:
The file content check would be better in the acceptance test.

Reply:
moved to the acceptance test and changed to `contain_file()` only